src/style/index.css

```
@@ -232,6 +232,12 @@ select {
   appearance: none;
 }
 
+select#theme option
```

I reckon this could be rolled out to all options, not just the theme picker.

```suggestion
option {
```

---

Yep, that's even better :)